js/githubresume.js

```diff
@@ -1,3 +1,5 @@
+if (window.location.protocol != 'https:') window.location.protocol = "https:";
+
```

Comment:
I don't think we should force https, and also it will cause problem for the developer run on local.